connectors/aws-cert/src/test/java/com/vmware/connectors/aws/cert/AwsCertControllerTest.java

```diff
@@ -124,6 +124,7 @@ void testRequestCardsSuccess(String lang, String responseFile) throws Exception
                 .collect(Collectors.joining())
                 .map(JsonNormalizer::forCards)
                 .block();
+        body = body.replaceAll("[a-z0-9]{40,}", "test-hash");
```

Comment:
The ```${backend_host}``` placeholder in ```hero_aws_cert_approval_url``` is replaced with mock provider host and port. So the hash value is getting changed every time. Because of this we cannot compare with actual hash value for ```AwsCertController```. I have to replace the actual hash value with ```test-hash```.

There is a similar issue with ```JiraConnector``` as well.